src/test/java/integration/AddressControllerIntegrationTest.java

```diff
@@ -0,0 +1,141 @@
+package integration;
+
+import org.json.simple.JSONObject;
+import org.junit.Test;
+
+import static io.restassured.RestAssured.*;
+import static org.hamcrest.Matchers.equalTo;
+import static org.hamcrest.Matchers.greaterThan;
+
+/**
+ * Created by Diarsid on 22.03.2017.
+ */
```

Comment:
Yeah, I've forgotten to disable it. Oleh has already pointed it while reviewing it.